Content.Server/Nutrition/Components/OpenableComponent.cs

```diff
@@ -38,9 +38,63 @@ public sealed partial class OpenableComponent : Component
     [DataField, ViewVariables(VVAccess.ReadWrite)]
     public LocId ClosedPopup = "drink-component-try-use-drink-not-open";
 
+    /// <summary>
+    /// Text to show in the verb menu for the "Open" action.
+    /// You may want to change this for non-drinks, i.e. "Peel", "Unwrap"
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public LocId OpenVerbText = "openable-component-verb-open";
+
+    /// <summary>
+    /// Text to show in the verb menu for the "Close" action.
+    /// You may want to change this for non-drinks, i.e. "Wrap"
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public LocId CloseVerbText = "openable-component-verb-close";
+
     /// <summary>
     /// Sound played when opening.
     /// </summary>
     [DataField]
     public SoundSpecifier Sound = new SoundCollectionSpecifier("canOpenSounds");
+
+    /// <summary>
+    /// Can this item be closed again after opening?
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public bool Closeable = false;
+
+    /// <summary>
+    /// Sound played when closing.
+    /// </summary>
+    [DataField]
+    public SoundSpecifier? CloseSound;
+
+    /// <summary>
+    /// If true, opening and reclosing this item will leave
+    /// evidence that it has been opened, shown in the
+    /// examined text. If false, no special text will be
+    /// displayed.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public bool Sealable = false;
```

Comment:
above

it also may be worth putting this in its own component or something

Reply:
Yeah, it probably would be cleaner to do a separate SealableComponent and have Openable raise an event on open. I'll work on that.